src/js/views/globals/search/patient-search.scss

```diff
@@ -33,12 +33,14 @@
 
 .patient-search__header {
   border-bottom: 1px solid $black-90;
+  margin: 0 8px;
+  padding-left: 16px;
 }
 
 .patient-search__picklist-item {
   color: $black-20;
   font-size: 16px;
-  padding: 4px 56px;
+  padding: 4px 64px;
```

Comment:
This aligns the search results items with the input text and help text content.

<img width="543" alt="Screen Shot 2022-09-08 at 12 06 38 PM" src="https://user-images.githubusercontent.com/35355575/189183189-9e97d701-ccc5-43c7-9367-0c21cef180db.png">